bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -136,7 +136,7 @@
       <developer-sidebar />
     </f7-panel>
 
-    <f7-view main v-show="ready" class="safe-areas" url="/" :master-detail-breakpoint="960" :animate="themeOptions.pageTransitionAnimation !== 'disabled'" />
+    <f7-view main v-if="ready" class="safe-areas" url="/" :master-detail-breakpoint="960" :animate="themeOptions.pageTransitionAnimation !== 'disabled'" />
```

Comment:
@ghys We should only initialize the f7-view after all data is loaded (i.e. ready is true). I have checked the app.vue code and found no reference to the f7-view, so I think this is fine.

Reply:
Alright, normally I never use `v-show` if I could use `v-if` instead, so there could be a reason why I did here in 2020 (https://github.com/openhab/openhab-webui/commit/6d12fe94fe2756b8966a908ace38d69b5dd85347) and just can't remember why atm. Let me sit on this one for a while...

Reply:
Let's just keep this last remark in mind and see how it goes.